src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsForm.tag

```diff
@@ -6,11 +6,13 @@
 <%@ tag import="teammates.common.util.FieldValidator" %>
 
 <%@ attribute name="fsForm" type="teammates.ui.template.FeedbackSessionsForm" required="true"%>
+<%@ attribute name="fsEnableEdit" %>
 
 <div class="well well-plain">
   <form class="form-group" method="post"
       action="${fsForm.formSubmitAction}"
-      id="form_feedbacksession">
+      id="form_feedbacksession"
+      data-<%= Const.ParamsNames.FEEDBACK_SESSION_ENABLE_EDIT %>="${fsEnableEdit}">
```

Comment:
> Maybe use c:if to only include the data attribute if the tag attribute is not set then.

Got it. Thats what I was suggesting [here](https://github.com/TEAMMATES/teammates/pull/7615#discussion_r129230073) haha.

Quick question: How do I check the if the property is set?